gulpfile.js

```diff
@@ -0,0 +1,90 @@
+var gulp = require('gulp');
```

Comment:
I have a question. I made the folder structure like below.

/dist
... /client
.../server
.../shared
.../test

Because, in the test code I wanted to import by relative path ( ex) ../server/model/user)
otherwise I needed to import absolute path( ex) dist-server/model/user).
In my opinion absolute path is not good for maintenance.

However, If testing with es6 is possible, I doesn't have to structured like my directories.
Is is possible to testing with es6 javascript code?